Lib9c.GraphQL/Objects/SheetObject.cs

```diff
@@ -0,0 +1,8 @@
+namespace Lib9c.GraphQL.Objects;
+
+public class SheetObject
```

Comment:
There is a reason you created a separate class to implement it?

Reply:
In Mimir's existing API, users could get a sheet in either CSV or JSON when they requested it. To adapt this to GraphQL, I defined a separate object.
Later on, I also expected this `SheetObject` to have a sheet address of type `Address` added to it.

> 미미르에서 기존에 제공하던 API에서는 사용자가 시트를 조회할 때, CSV나 JSON 중 원하는 형태로 받을 수 있었습니다. 이를 GraphQL에 적용하기 위해서 별도의 오브젝트를 정의했습니다.
> 이후에는 이 `SheetObject`에 `Address` 타입의 시트 주소가 추가되는 걸 기대하기도 했습니다.